examples/monitor-with-thanos/tidb-monitor.yaml

```diff
@@ -8,6 +8,9 @@ spec:
   thanos:
     baseImage: thanosio/thanos
     version: v0.17.2
+    objectStorageConfig:
+      key: objectstorage.yaml
+      name: thanos-objectstorage-hbase
```

Comment:
Why adding this? From the doc https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/aggregate-multiple-cluster-monitor-data here, this config should not be included by default.